### DISCUSSION TITLE
Python/C/C++

### DISCUSSION THREAD

**Discussion by hili94** [WeAreMahsaAmini/Core#309] [Introductions]
Hi, I am Hilma. I'm a firmware engineer and native English/Farsi speaker. I have 6 years of programming experience though not in the web, but now's a good time to start learning..

**Comment by ehsaaanes** [comment]
Hi Hilma, I'm in search of people who are willing to join me in developing a social media app as a start-up project. 
Would you like to know about it? 
Ehsan

**Comment by tcf-admin** [comment]
Hi @hili94, 

We have some projects which you might find interesting. 
- The backend of forDem, an e-democracy platform (ported from Ruby and extended in C#) https://github.com/tcfev/mastodon-csharp
issues:
https://github.com/tcfev/forDem/issues/158
- a plan to extend and then deploy openMesh in scale: https://openwrt.org/toh/openmesh/om2p 
- and secret project (contact)

Hopefully you're still around and motivated :-)
